src/main/java/net/minecraftforge/common/config/Configuration.java

```diff
@@ -80,9 +89,16 @@
 
     public Configuration(){}
 
+    @Deprecated
+    public Configuration(CharSource staticSource, CharSink staticSink){
```

Comment:
Ok. I'm keeping char source & sink, i'll unmark them deprecated.
But still, i'm thinkg of keeping Buffered and Stream load & save methods. Like that, youcan use char source and sink to load & save using load() and save(), but if you want, you can use streams or buffered readers & writers.